src/components/Panel/Actions/ActionDetails.tsx

```diff
@@ -0,0 +1,126 @@
+import { Currency, InformationIcon } from "@/components";
+import type { OracleQueryUI } from "@/types";
+import styled from "styled-components";
+import { Link, Text } from "../style";
+
+export function ActionDetails({
+  bond,
+  reward,
+  formattedLivenessEndsIn,
+  tokenAddress,
+  chainId,
+}: OracleQueryUI) {
+  const hasReward = reward !== null;
+
+  return (
+    <Wrapper>
+      <ActionWrapper>
+        <ActionText>
+          Bond
+          <InformationIcon content={bondInformation} />
+        </ActionText>
+        <ActionText>
+          <Currency address={tokenAddress} chainId={chainId} value={bond} />
+        </ActionText>
+      </ActionWrapper>
+      {hasReward && (
+        <ActionWrapper>
+          <ActionText>
+            Reward
+            <InformationIcon content={rewardInformation} />
+          </ActionText>
+          <ActionText>
+            <Currency address={tokenAddress} chainId={chainId} value={reward} />
+          </ActionText>
+        </ActionWrapper>
+      )}
+      <ActionWrapper>
+        <ActionText>
+          Challenge period ends in
+          <InformationIcon content={livenessInformation} />
+        </ActionText>
+        <ActionText>{formattedLivenessEndsIn}</ActionText>
+      </ActionWrapper>
+    </Wrapper>
+  );
+}
+
+// todo: @sean update copy
+const bondInformation = (
+  <>
+    <p>
+      Every request to UMA&apos;s Optimistic Oracle includes bond settings that
+      specify the size of the bond that proposers (and disputers) are required
+      to post.
+    </p>
+    <br />
+    <p>The minimum bond is the final fee for a given bond token.</p>
+    <br />
+    <Link
+      href="https://docs.uma.xyz/developers/setting-custom-bond-and-liveness-parameters"
+      target="_blank"
+    >
+      Learn more
+    </Link>
+  </>
+);
+
+// todo: @sean update copy
+const rewardInformation = (
+  <>
+    <p>
+      Lorem ipsum dolor sit, amet consectetur adipisicing elit. Illum, beatae.
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quod.
```

Comment:
There was a recent pr from sean for copy, may want to check if this was included. If not someone else will need to make change

Reply:
I re-added it when I moved it 👍